pkg/securitycontext/util.go

```diff
@@ -47,26 +44,6 @@ func HasCapabilitiesRequest(container *v1.Container) bool {
 	return len(container.SecurityContext.Capabilities.Add) > 0 || len(container.SecurityContext.Capabilities.Drop) > 0
 }
 
-const expectedSELinuxFields = 4
-
-// ParseSELinuxOptions parses a string containing a full SELinux context
-// (user, role, type, and level) into an SELinuxOptions object.  If the
-// context is malformed, an error is returned.
-func ParseSELinuxOptions(context string) (*v1.SELinuxOptions, error) {
```

Comment:
Given that it's a public method, could it be a breaking change?

Reply:
Just a query regarding this, How do we parse SELinux context roles, user etc after that ? Do we have other usecase for this ?

Reply:
> Given that it's a public method, could it be a breaking change?

we do not advertise kubernetes/kubernetes as a utility library. if someone is depending on this, it's a simple enough function to copy into their code